mixes/helpers/flatten.js

```diff
@@ -29,7 +29,7 @@ function flattenChannels (channel) {
     const { channels = [], clips = [] } = next
     return {
       [next.id]: {
-        ...next,
+        ...omit(next, 'channels', 'clips'),
```

Comment:
@ahdinosaur  can you sanity check this change? i didnt follow the logic all the way through but this seemed right ish. the tests were failing because we were including `channels` and `clips` in the resulting `flatMix`'s nestedChannels. this seemed like a real bug.

i  wanted to use `omit` instead of `pick` to ensure we pick up other properties of the channels, eg `type`. updated test to test for that as well. does this seem right to you?